### DISCUSSION TITLE
[Detector Support]: Openvino - No supported devices found since the addition of the environment variable OPENCV_FFMPEG_LOGLEVEL in Dockerfile

### DISCUSSION THREAD

**Discussion by ubawurinna** [blakeblackshear/frigate#14873] [Detector Support]
### Checklist

- [X] I have updated to the latest available Frigate version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [incognito mode](https://www.computerworld.com/article/1719851/how-to-go-incognito-in-chrome-firefox-safari-and-edge.html) to rule out problems with any third party extensions or plugins I have installed.

### Describe the problem you are having

The problem started with commit d7935ab.
No openvino compatible device is detected and Frigate keeps restarting.

### Steps to reproduce

Use GPU with OpenVino (Intel 6th gen cpu is used in my case)

### Version

0.15.0-d7935ab

### In which browser(s) are you experiencing the issue with?

Unrelated

### Frigate config file

```yaml
ffmpeg:
  hwaccel_args: preset-vaapi

detectors:
  ov:
    type: openvino
    device: GPU
```

### docker-compose file or Docker CLI command

```yaml
-
```

### Relevant Frigate log output

```shell
Process detector:ov:
  Traceback (most recent call last):
    File "/usr/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
      self.run()
    File "/opt/frigate/frigate/util/process.py", line 41, in run_wrapper
      return run(*args, **kwargs)
    File "/usr/lib/python3.9/multiprocessing/process.py", line 108, in run
      self._target(*self._args, **self._kwargs)
    File "/opt/frigate/frigate/object_detection.py", line 121, in run_detector
      object_detector = LocalObjectDetector(detector_config=detector_config)
    File "/opt/frigate/frigate/object_detection.py", line 68, in __init__
      self.detect_api = create_detector(detector_config)
    File "/opt/frigate/frigate/detectors/__init__.py", line 18, in create_detector
      return api(detector_config)
    File "/opt/frigate/frigate/detectors/plugins/openvino.py", line 40, in __init__
      self.interpreter = self.ov_core.compile_model(
    File "/usr/local/lib/python3.9/dist-packages/openvino/runtime/ie_api.py", line 543, in compile_model
      super().compile_model(model, device_name, {} if config is None else config),
  RuntimeError: Exception from src/inference/src/cpp/core.cpp:121:
  Check '!m_device_map.empty()' failed at src/plugins/intel_gpu/src/plugin/plugin.cpp:379:
  [GPU] Can't get OPTIMIZATION_CAPABILITIES property as no supported devices found or an error happened during devices query.
  [GPU] Please check OpenVINO documentation for GPU drivers setup guide.
```

### Relevant go2rtc log output

```shell
-
```

### Operating system

Debian

### Install method

Docker Compose

### Network connection

Wireless

### Camera make and model

tapo c510w

### Screenshots of the Frigate UI's System metrics pages

-

### Any other information that may be helpful

Setting OPENCV_FFMPEG_LOGLEVEL to no value or to any value has no effect
None of Frigate 0.15's new features are used

**Comment by NickM-27** [comment]
it is unlikely these are related. If anything, it is possible that an intel package was updated around the same time but we don't have any way of undoing that. Looks like we will just need to wait until intel fixes this bug

**Reply by ubawurinna** [reply]
You're right, libigxxx.so files have changed from previous images

Previous image 1234003
```
/usr/lib/x86_64-linux-gnu/libiga64.so.1.0.17537.24
/usr/lib/x86_64-linux-gnu/libigc.so.1.0.17537.24
/usr/lib/x86_64-linux-gnu/libigdfcl.so.1.0.17537.24
```

Image d7935ab
```
/usr/lib/x86_64-linux-gnu/libiga64.so.1.0.17791.16
/usr/lib/x86_64-linux-gnu/libigc.so.1.0.17791.16
/usr/lib/x86_64-linux-gnu/libigdfcl.so.1.0.17791.16

```

**Reply by NickM-27** [reply]
I have put up a PR to pin the intel driver version at the previous one, hopefully that fixes the issue

**Reply by jurasjo** [reply]
Unfortunately, this PR didn't fix the issue.

**Reply by NickM-27** [reply]
Hmm, we'll need to know what package changed version

**Reply by ubawurinna** [reply]
commit a68c7f4 solved the issue

**Reply by ubawurinna** [reply]
I've done a few tests and the **intel-opencl-icd** package is the faulty one.
24.35.30872 works
24.39.31294 doesn't work
All other package can be set to the last version

**Reply by jurasjo** [reply]
Yes, I can also confirm https://github.com/blakeblackshear/frigate/commit/a68c7f4ef859cfb15cb2275865dd46500e17e148 works.

**Reply by hawkeye217** [reply]
> I've done a few tests and the **intel-opencl-icd** package is the faulty one.
> 
> 24.35.30872 works
> 
> 24.39.31294 doesn't work
> 
> All other package can be set to the last version

Thanks!

**Comment by mattfox27** [comment]
was there a fix for this?  I have the same issue

**Reply by hawkeye217** [reply]
Yes, the fix was committed several months ago. If you are having an issue, go ahead and open your own support discussion with all of the requested information.
